src/java.base/share/classes/java/lang/System.java

```diff
@@ -324,6 +326,16 @@ private static void checkIO() {
     private static native void setOut0(PrintStream out);
     private static native void setErr0(PrintStream err);
 
+    // Remember original System.err. setSecurityManager() warning goes here
+    private static volatile @Stable PrintStream originalErrStream = null;
```

Comment:
I'd probably use "initial" rather than "original" here but okay. You can drop setting it to null as that is its default value.

Reply:
Will do. Thanks.